packages/runtime/src/binding/target-observer.ts

```diff
@@ -31,12 +31,10 @@ function setValue(this: BindingTargetAccessor, newValue: Primitive | IIndexable,
 }
 
 function flush(this: BindingTargetAccessor, flags: LifecycleFlags): void {
-  if (flags & LifecycleFlags.doNotUpdateDOM) {
-    if (DOM.isNodeInstance(this.obj)) {
-      // re-queue the change so it will still propagate on flush when it's attached again
-      this.lifecycle.enqueueFlush(this).catch(error => { throw error; });
-      return;
-    }
+  if (flags & LifecycleFlags.doNotUpdateDOM && DOM.isNodeInstance(this.obj)) {
```

Comment:
```suggestion
  if ((flags & LifecycleFlags.doNotUpdateDOM) && DOM.isNodeInstance(this.obj)) {
```
I know this doesn't matter functionally, but bitwise operators aren't particularly well known by most developers so I'd like to keep the extra clarify.